install/install.sh

```diff
@@ -148,6 +279,61 @@ uninstall_purge() {
         docker rmi sodafoundation/dashboard:$DASHBOARD_RELEASE_VERSION
         echo -e "\033[32m Dashboard image has been removed. \033[m"
     fi
+    
+    # Purge Prometheus Image
+    if [ ! "$(docker images | grep prom/prometheus)" ]
+    then
+        echo "Prometheus image is not available."        
+    else
+        echo -e "\e[1;31m Removing prometheus image. \e[0m" 
+        docker rmi prom/prometheus:$prometheus_image_tag
+        echo -e "\033[32m Prometheus image has been removed. \033[m"
+    fi
+
+    # Purge Alertmanager Image
+    if [ ! "$(docker images | grep prom/alertmanager)" ]
+    then
+        echo "Alertmanager image is not available."        
+    else
+        echo -e "\e[1;31m Removing alertmanager image. \e[0m" 
+        docker rmi prom/alertmanager:$alertmanager_image_tag
+        echo -e "\033[32m Alertmanager image has been removed. \033[m"
+    fi
+
+    # Purge Grafana Image
+    if [ ! "$(docker images | grep grafana/grafana)" ]
+    then
+        echo "Grafana image is not available."        
+    else
+        echo -e "\e[1;31m Removing grafana image. \e[0m" 
+        docker rmi grafana/grafana:$grafana_image_tag
+        echo -e "\033[32m Grafana image has been removed. \033[m"
+    fi
+
+    # Purge Prometheus Volume
+    if [ ! "$(docker volume ls | grep prometheus_data)" ]
+    then
+        echo "Prometheus volume is not available."        
+    else
+        echo -e "\e[1;31m Removing prometheus volume. \e[0m" 
+        docker volume rm $(docker volume ls | grep prometheus_data)
+        echo -e "\033[32m Prometheus volume has been removed. \033[m"
+    fi
+
+    # Purge Grafana Volume
+    if [ ! "$(docker volume ls | grep grafana_data)" ]
+    then
+        echo "Grafana volume is not available."        
+    else
+        echo -e "\e[1;31m Removing grafana volume. \e[0m" 
+        docker volume rm $(docker volume ls | grep grafana_data)
+        echo -e "\033[32m Grafana volume has been removed. \033[m"
+    fi
+
+    # Deleting the SRM Toolchain work folder
+    echo -e "\e[1;31m Deleting the SRM toolchain workfolder. \e[0m" 
+    rm -Rf /opt/srm-toolchain/
```

Comment:
/opt/srm-toolchain/ needs to be configurable. All parameters in SODA installer ansible/group_vars/srm_toolschain.yml needs to be taken as input for the script

Reply:
Made the work directory configurable. User needs to export this as an environment variable or the default value is picked.